[WIP] Retire VS Container Tools article and forward to ASP.NET Core documentation (#37420)
dotnet/AspNetCore.Docs: aspnetcore/host-and-deploy/docker/index.md

[.NET Docker samples](https://github.com/dotnet/dotnet-docker/tree/main/samples)
Samples and guidance that demonstrate how to use .NET and Docker for development, testing and production.

[Visual Studio Container Tools](/visualstudio/containers/add-container-support)  

> Discover how Visual Studio supports building, debugging, and running ASP.NET Core apps on Docker for Windows. Both Windows and Linux containers are supported.

Reviewer: ```suggestion
Discover how Visual Studio supports building, debugging, and running containerized apps, including an ASP.NET Core example. Both Windows and Linux containers are supported.
```
